tests/countries/test_united_states.py

```diff
@@ -2368,8 +2369,8 @@ def test_l10n_th(self):
             (
                 "2022-11-25",
                 (
-                    "วันครอบครัว; วันประธานาธิบดี; วันหยุดประจำรัฐ; วันหลังวันขอบคุณพระเจ้า; "
-                    "วันอนุรักษ์มรดกชนพื้นเมืองอเมริกัน; วันเกิดลิงคอล์น; "
+                    "วันครอบครัว; วันประธานาธิบดี; วันมรดกชนพื้นเมืองอเมริกัน; วันหยุดประจำรัฐ; "
+                    "วันหลังวันขอบคุณพระเจ้า; วันอนุรักษ์มรดกชนพื้นเมืองอเมริกัน; วันเกิดลิงคอล์น; "
```

Comment:
Since the `th` translation for "Native American Heritage Day" is the same as the one for the existing "American Indian Heritage Day", there's no extra entries here for `test_l10n_th`
```suggestion
                    "วันครอบครัว; วันประธานาธิบดี; วันหยุดประจำรัฐ; วันหลังวันขอบคุณพระเจ้า; "
                    "วันอนุรักษ์มรดกชนพื้นเมืองอเมริกัน; วันเกิดลิงคอล์น; "
```